src/main/xerus/monstercat/MonsterUtilities.kt

```diff
@@ -310,4 +314,64 @@ class MonsterUtilities(checkForUpdate: Boolean): VBox(), JFXMessageDisplay {
 		}
 	}
 	
+	/** Shows a new window with an ImageView of the requested [coverUrl]
+	 * [title] Title of the window, only useful when decorated
```

Comment:
please use `@param` instead of links

Reply:
ce50572db5b3eda667478e18f6d1c9fd1d0584b0